src/OpenRiaServices.Tools.CodeGenTask/OpenRiaServices.Tools.CodeGenTask.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```

Comment:
Currently only net6 is used, it might make sense to wait with net7 build until we know it is needed

Reply:
```suggestion
    <TargetFramework>net6.0</TargetFramework>
```